frontend/package.json

```diff
@@ -1,9 +1,11 @@
 {
   "name": "frontend",
-  "version": "1.1.0-25.10",
+  "version": "1.1.0-25.11",
   "private": true,
   "dependencies": {
     "@bcgov/bc-sans": "1.0.1",
+    "@mdi/js": "^6.6.95",
```

Comment:
Hmm, we want to be careful about this one I think. For one react-icons (should) contain the material design icons
https://react-icons.github.io/react-icons/icons?name=md
Icon libraries can be a bit on the big side so I wouldn't want us to bring this in unless:
1. the library is small **or** 
2. only adds icons that we are actively using to the build.
3. **and** the icons being added aren't part of react-icons.

Reply:
The reason I added this one is because that is the icon on the mockups. I think there is a version of it as an svg. But maybe we want to stick with some icon packs.

Reply:
yeah, like I said, all of the md icons should be part of react-icons, can you confirm if it is/isn't?

Reply:
This is the one I found that matches the mockup:
https://github.com/bcgov/PSP/pull/1205